src/BenchmarksApps/TodosApi/appsettings.Development.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "Microsoft.AspNetCore": "Debug"
+    }
+  },
+  "Authentication": {
+    "Schemes": {
+      "Bearer": {
+        "ValidAudiences": [
+          "http://localhost:5054"
+        ],
+        "ValidIssuer": "dotnet-user-jwts"
+      }
+    }
+  }
```

Comment:
Would it make sense to not duplicate this section, since it is in the appsettings.json?